### DISCUSSION TITLE
Meeting page not working

### DISCUSSION THREAD

**Discussion by eflashlogistics** [code4recovery/12-step-meeting-list#918] [Q&A] [answered]
Greetings

I just reinstalled the plugin and now when I goto my meeting page it doesnt show up "page not found"

Any ideas?

Thanks

http://lucasg11.sg-host.com/meetings/

**Comment by lovethecodelabs** [comment, marked as answer]
Resave permalinksOn Sep 11, 2022 9:59 AM, eflashlogistics ***@***.***> wrote:
Greetings
I just reinstalled the plugin and now when I goto my meeting page it doesnt show up "page not found"
Any ideas?
Thanks
http://lucasg11.sg-host.com/meetings/

**Comment by eflashlogistics** [comment]
ok reset permalinks and it worked

**Comment by lovethecodelabs** [comment]
Awesome!On Sep 11, 2022 10:25 AM, eflashlogistics ***@***.***> wrote:
ok reset permalinks and it worked
